src/tracks/playabletrack/wavetrack/ui/WaveTrackAffordanceControls.cpp

```diff
@@ -52,6 +57,32 @@ class WaveTrackAffordanceHandle final : public AffordanceHandle
     }
 };
 
+class SetWaveClipNameCommand : public AudacityCommand
+{
+public:
+    static const ComponentInterfaceSymbol Symbol;
+
+    ComponentInterfaceSymbol GetSymbol() override
+    {
+        return Symbol;
+    }
+    void PopulateOrExchange(ShuttleGui& S) override
```

Comment:
Is this your first experience writing some ShuttleGui code?

Reply:
Sort of